src/components/overview/addToCart/AddToCart.jsx

```diff
@@ -5,19 +5,13 @@ import json from '../styleSample.json';
 
 const AddToCart = (props) => {
 
-  var options = json.results;
-  var currentSkus = options[0].skus; //Object of containing skus;
-
+  var currentSkus = props.skus;
```

Comment:
For instance if skus is the only prop that you have passed into this component then you could cut out a whole line of code by decomposing it on line 6! Just food for thought or another potential way of doing it.